examples/stress-test/run.py

```diff
@@ -0,0 +1,202 @@
+from zenml.config.docker_settings import PythonPackageInstaller
+from zenml.integrations.kubernetes.flavors.kubernetes_orchestrator_flavor import (
+    KubernetesOrchestratorSettings,
+)
+from zenml.integrations.kubernetes.pod_settings import KubernetesPodSettings
+
+kubernetes_settings = KubernetesOrchestratorSettings(
+    pod_startup_timeout=600,
+    pod_settings=KubernetesPodSettings(
+        resources={
+            "requests": {"cpu": "100m", "memory": "300Mi"},
+            "limits": {"memory": "350Mi"},
+        },
+        node_selectors={"pool": "workloads"},
+        tolerations=[
+            {"key": "pool", "operator": "Equal", "value": "workloads", "effect": "NoSchedule"}
+        ],
+        env=[{"name": "ZENML_LOGGING_VERBOSITY", "value": "debug"}],
+    ),
+    orchestrator_pod_settings=KubernetesPodSettings(
+        resources={
+            "requests": {"cpu": "100m", "memory": "300Mi"},
+            "limits": {"memory": "350Mi"},
+        },
+        node_selectors={"pool": "workloads"},
+        tolerations=[
+            {"key": "pool", "operator": "Equal", "value": "workloads", "effect": "NoSchedule"}
+        ],
+    ),
+)
+
+docker_settings = DockerSettings(
+    python_package_installer=PythonPackageInstaller.UV,
+    parent_image="public.ecr.aws/f7g3v0f1/zenml:stefan",
+    #prevent_build_reuse=True,
+)
+settings = {"docker": docker_settings, "orchestrator": kubernetes_settings}
+
+
+@step
+def load_step(
+    duration: int,
+    sleep_interval: float,
+) -> Dict[str, Any]:
+    client = Client()
+    start_time = time.time()
+    operations = 0
+
+    print("Starting API calls...")
+    while time.time() - start_time < duration:
+        # Perform various API operations
+        print("Listing pipeline runs...")
+        client.list_pipeline_runs()
+        print("Listing stacks...")
+        client.list_stacks()
+        print("Listing stack components...")
+        client.list_stack_components()
+        print("Listing service connectors...")
+        client.list_service_connectors()
+
+        operations += 4
+        if sleep_interval > 0:
+            print(f"Sleeping for {sleep_interval} seconds...")
+            time.sleep(sleep_interval)
+
+    return {
+        "operations": operations,
+        "duration": duration,
+    }
+
+
+# The report results step is beefier than the load step because it has to fetch
+# all the artifacts from the run.
+kubernetes_settings = KubernetesOrchestratorSettings(
+    pod_settings=KubernetesPodSettings(
+        resources={
+            "requests": {"cpu": "100m", "memory": "500Mi"},
+            "limits": {"memory": "600Mi"},
+        },
+        node_selectors={"pool": "workloads"},
+        tolerations=[
+            {"key": "pool", "operator": "Equal", "value": "workloads", "effect": "NoSchedule"}
+        ],
+        env=[{"name": "ZENML_LOGGING_VERBOSITY", "value": "debug"}],
+    ),
+)
+
+settings = {"docker": docker_settings, "orchestrator": kubernetes_settings}
```

Comment:
Maybe I'm missing something, but this simply overrides the initial declaration of the `settings`?

Reply:
Yes, I need the last step to be beefier

Reply:
All steps use these settings, because they're defined on the pipeline though?

Reply:
True. I missed that, thank you!